projects.md

```diff
@@ -0,0 +1,70 @@
+# Projects
+
+## Sales Prospect & Internal Proposals
+
+While the Business Planning WG is mainly in charge of sales lead generation,
+all members may pursue leads as they come up.
+These may be a client or a grant opportunity.
```

Comment:
```suggestion

```

Don't think this adds values and just imposes a potentially limiting categorization